acceptance/acceptance_test.go

```diff
@@ -486,6 +491,15 @@ func runTest(t *testing.T,
 		tmpDir, err = os.MkdirTemp(tempDirBase, "")
 		require.NoError(t, err)
 		t.Logf("Created directory: %s", tmpDir)
+	} else if WorkspaceTmpDir {
+		// If the test is being run on DBR, auth is already configured
+		// by the dbr_runner notebook by reading a token from the notebook context and
```

Comment:
the notebook will be added in a followup PR: https://github.com/databricks/cli/pull/3453